### ISSUE TITLE
Accessibility - Resources Screen - Three Links Not Read by VoiceOver as Links

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OCIO-DOLSafety-iOS#27]
The LaborNet, Agency Internets, and OWSH links are not being announced as a link, and the user is not able to tap them and go to the website.

Please see the attached screenshot.

**NOTE**: Black oval is the focus ring for VoiceOver, so people know where they are on the screen.

![safetynet - resources screen - three links not active links](https://user-images.githubusercontent.com/10423487/61553388-53a13b80-aa28-11e9-862d-5920ba1e40d1.png)

**Comment by aanthonyDOL** [comment]
Per Requirements these aren't to function as links ; due to the fact that they link to internal websites. Furthermore the apps current functionality doesn't support opening links to internal websites. I've added Voice over to call them out as links.

**Comment by jfc3-dol** [comment]
Wouldn't anyone able to download the application have access to internal links such as LaborNet?

Thanks for telling the user they are links in VoiceOver.
